ucdlib-special/includes/main.php

```diff
@@ -24,6 +25,18 @@ public function __construct(){
     // config values. slugs and what not.
     $this->config = new UCDLibPluginSpecialConfig();
 
+    // capabilities for each custom post type
```

Comment:
move to config file. also you have to make sure these capabilities actually get assigned to the `collection_manager` role on plugin activation.

Reply:
Will do